estimator/top_level.py

```diff
@@ -0,0 +1,73 @@
+from .conf import red_cost_model_default, red_shape_model_default
+from .lwe_dual import dual, dual_hybrid, dual_mitm_hybrid
+from .lwe_primal import primal_usvp, primal_hybrid, primal_bdd
+from .gb import arora_gb
+from .lwe_brute_force import exhaustive_search, mitm
+
+
+def estimate_lwe(params,
```

Comment:
I've got an unpushed version of this, will merge the two

Reply:
Feel free to completely overwrite -- this was a very quick first effort.